src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/DescriptorsValidation.feature.disabled

```diff
@@ -0,0 +1,285 @@
+            Given the system has this codeValue "Deaf-Blindness"
+             When a PUT request is made to "/ed-fi/disabilityDescriptors/{id}" with
+                  """
+                  {
+                      "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                      "codeValue": "Deaf-Blindness",
+                      "shortDescription": "Deaf-Blindness",
+                      "description": "Deaf-Blindness"
+                      "effectiveBeginDate": "2024-07-24",
+                      "effectiveEndDate" "2025-12-31"
+                  }
+                  """
+             Then it should respond with 204
+
+        @ignore
+        Scenario: 07 / Ensure clients can retrieve a descriptor by requesting through a valid codeValue
+            Given the system has this codeValue "Deaf-Blindness"
+             When a GET request is made to "/ed-fi/disabilityDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  [
+                      {
+                          "id": "{id}",
+                          "codeValue": "Deaf-Blindness",
+                          "description": "Deaf-Blindness",
+                          "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                          "shortDescription": "Deaf-Blindness",
+                      }
+                  ]
+                  """
+
+        @ignore
+        Scenario: 08 / Ensure clients cannot retrieve a descriptor by requesting through a non existing codeValue
+             When a GET request is made to "/ed-fi/disabilityDescriptors?codeValue=Test"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  []
+                  """
+
+        @ignore
+        Scenario: 09 / Ensure clients can retrieve a descriptor by requesting through a valid namespace
+            Given the system has this namespace "uri://ed-fi.org/DisabilityDescriptor"
+             When a GET request is made to "/ed-fi/disabilityDescriptors/{id}"
+             Then it should respond with 200
+              And the response body is
+                  """
+                  [
+                      {
+                          "id": "{id}",
+                          "codeValue": "Deaf-Blindness",
+                          "description": "Deaf-Blindness",
+                          "namespace": "uri://ed-fi.org/DisabilityDescriptor",
+                          "shortDescription": "Deaf-Blindness",
+                      }
+                  ]
+                  """
+
+        @ignore
+        Scenario: 10 / Ensure clients cannot retrieve a descriptor by requesting through a non existing namespace
+             When a GET request is made to "/ed-fi/disabilityDescriptors?namespace=uri://ed-fi.org/DisabilityDescriptorFake"
```

Comment:
`uri://ed-fi.org/DisabilityDescriptorFake` has special characters in it that need to be [URL Encoded](https://www.urlencoder.io/) when used in a URL. Thus, this should be `namespace=uri%3A%2F%2Fed-fi.org%2FDisabilityDescriptorFake`.

Reply:
Fixed